src/app-layout/visual-refresh-toolbar/exports.tsx

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export { AppLayoutDrawerImplementation as AppLayoutDrawer } from './drawer';
+export { AppLayoutNavigationImplementation as AppLayoutNavigation } from './navigation';
+export { AppLayoutNotificationsImplementation as AppLayoutNotifications } from './notifications';
+export {
+  AppLayoutSplitPanelDrawerBottomImplementation as AppLayoutSplitPanelDrawerBottom,
+  AppLayoutSplitPanelDrawerSideImplementation as AppLayoutSplitPanelDrawerSide,
+} from './split-panel';
+export { AppLayoutToolbarImplementation as AppLayoutToolbar } from './toolbar';
```

Comment:
The new widgetization strategy is here. 

Widgetizing the whole app layout appeared to be a breaking change, so we widgetize all these parts individually instead